src/ngraph/frontend/onnx_import/op/gemm.cpp

```diff
@@ -73,15 +71,22 @@ namespace ngraph
                     input_a = ngraph::builder::opset1::flatten(input_a, 1);
                     input_b = ngraph::builder::opset1::flatten(input_b, 1);
 
-                    auto matmul_node = std::make_shared<default_opset::MatMul>(input_a, input_b);
+                    std::shared_ptr<ngraph::Node> last_node =
+                        std::make_shared<default_opset::MatMul>(input_a, input_b);
+
+                    if (alpha != 1)
+                    {
+                        last_node = std::make_shared<default_opset::Multiply>(
+                            last_node,
+                            default_opset::Constant::create(
```

Comment:
Let's keep this Constant assigned to `alpha_node`, a compiler will optimize the name away, but it will be clearer for the next programmer what's going on.

Reply:
Done.